android/app/src/main/java/com/pepperoniapptemplate/MainActivity.java

```diff
@@ -1,6 +1,7 @@
 package com.pepperoniapptemplate;
 
 import com.facebook.react.ReactActivity;
+import com.oblador.vectoricons.VectorIconsPackage;
```

Comment:
Actually this import needs to go in the `MainApplication.java` and this 
```
new VectorIconsPackage()
```
needs to be added inside the list of packages